aleph/settings.py

```diff
@@ -154,7 +159,7 @@ def __init__(self) -> None:
         )
 
         # Publishing network diagram embeds
-        self.REACT_FTM_URL = "https://cdn.jsdelivr.net/npm/@alephdata/react-ftm@latest/dist/react-ftm-embed.js"  # noqa
```

Comment:
Was this a conscious decision?

Reply:
No. `black` must have stripped out the `noqa`. I added it back.